### DISCUSSION TITLE
Alguien me puedes responder una Duda

### DISCUSSION THREAD

**Discussion by DyFrabian** [Launch-X-Latam/MisionFrontEnd#365] [Q&A]
¿Se envia la solicitud . Para el ingreso a Backend y se espera a qe te acepten o el ingreso es inmefiato?

**Comment by Gilberto-Guzman** [comment]
Hola @DyFrabian 
En tu correo electrónico te debe de llegar un enlace el cual te permita mandar una solicitud de ingreso. Ya los administradores te aceptaran posteriormente. Cualquier cosa no dudes en preguntar va? :D
